rca/utils/blocks.py

```diff
@@ -52,6 +54,47 @@ class Meta:
         icon = "image"
 
 
+class AccordionBlockWithTitle(blocks.StructBlock):
+    heading = blocks.CharBlock(
+        help_text="A large heading diplayed next to the block", required=False
+    )
+    preview_text = blocks.CharBlock(
+        help_text="The text to display when the accordion is collapsed", required=False
+    )
+    body = blocks.RichTextBlock(
+        help_text="The content shown when the accordion expanded",
+        features=["h2", "h3", "bold", "italic", "image"],
```

Comment:
will the features differ a lot? Maybe we should set them at https://docs.wagtail.io/en/v2.5/advanced_topics/settings.html#wagtailadmin-rich-text-editors or have a global list somewhere.

Reply:
Yea I did think about this, we don't have designs for other page models yet so it's hard to choose, I went with being explicit until I know more.